x-pack/plugins/maps/public/elasticsearch_geo_utils.js

```diff
@@ -400,6 +397,16 @@ export function getBoundingBoxGeometry(geometry) {
   return formatEnvelopeAsPolygon(extent);
 }
 
+export function makeGeotileGridDsl(field, bounds, precision, options = {}) {
```

Comment:
Is `makeGeotileGridDsl` even needed? Maybe best for consumers to just use `makeESBbox` themselves and create DSL since this wrapper is not doing much